apps/settings/style/settings.css

```diff
@@ -287,7 +287,8 @@ body[data-filter-by="connectivity"] #root > div [data-category="connectivity"] {
 .disabled label,
 .disabled select,
 .disabled gaia-checkbox,
-.disabled gaia-switch {
+.disabled gaia-switch,
+gaia-switch[disabled] {
```

Comment:
We can, but I would prefer to wait to do this until we are using the behavior in one more app if that is ok (and I will gladly move it into the component stylesheets).